cllc-public-app/ClientApp/src/app/models/sep-city.model.ts

```diff
@@ -0,0 +1,18 @@
+export class SepServiceArea {
```

Comment:
Please change the name of this class. I suspect this will break the ng build as there is a conflict in class names.